impeller/README.md

```diff
@@ -207,18 +207,31 @@ under the top-level `<dict>` tag:
 Impeller is in preview on Android.
 
 To your `AndroidManifest.xml` file, add under the `<application>` tag:
-```
+
+```xml
   <meta-data
     android:name="io.flutter.embedding.android.EnableImpeller"
     android:value="true" />
 ```
 
+Impeller will use Vulkan on Android by default when opted into. Where Vulkan
+is unavalable, Impeller will fallback to Skia. However, Impellers OpenGL backend
+is well under construction. To try that with your application, add the following
+under the `<application>` tag:
+
+```xml
+  <meta-data
+    android:name="io.flutter.embedding.android.ImpellerBackend"
+    android:value="opengles" />
+```
```

Comment:
Add note that this can only be opted into in debug/profile builds:

https://github.com/flutter/engine/blob/main/shell/platform/android/flutter_main.cc#L245-L256

Reply:
oh wow, I just learned this. wilco.

Reply:
Done.